exercises/bowling/.gitignore

```diff
@@ -0,0 +1 @@
+.cache/
```

Comment:
I would advise not to include this in repo `.gitignore`. You can your local one for this purpose